yarn-project/run_nargo_fmt.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Find all Nargo.toml files and run 'nargo fmt'
+find . -name "Nargo.toml" | while read -r file; do
+    # Extract the directory from the file path
+    dir=$(dirname "$file")
+    
+    # Change into the directory
+    cd "$dir" || exit
+
+    # Run 'nargo fmt' in the directory and pass in the input param 1
+    nargo fmt $1
```

Comment:
Should it run the nargo in aztec-packages/noir/target/release/nargo ? just in case the one in user's path is different than the generated during bootstrap

Reply:
Good catch. Addressed in https://github.com/AztecProtocol/aztec-packages/pull/3803/commits/56f4793e7a2145f76dadf071a2c4800f325567dd